apps/publish.sh

```diff
@@ -95,7 +97,18 @@ status "Publishing apps; version: ${APPS_VERSION}, Target tag: ${TARGET_TAG}"
     --new-targets-file="${ARCHIVE}/targets-created.json" \
     2>&1 | indent
 
-cp "${TUF_REPO}/roles/unsigned/targets.json" "${ARCHIVE}/targets-after.json"
+cp "${TUF_REPO}/roles/unsigned/targets.json" "${ARCHIVE}/targets-before1.json"
+cp "${ARCHIVE}/targets-before.json" "${TUF_REPO}/roles/unsigned/targets.json"
+
+
+/usr/local/bin/dind "${HERE}/publish_apps_to_ostree.py" \
+  --factory "${FACTORY}" \
+  --token "$(cat "${SECRETS}/osftok")" \
+  --cred-arch "${CREDS_ARCH_UPDATED}" \
+  --targets-file "${TUF_REPO}/roles/unsigned/targets.json" \
+  --targets-to-publish "${ARCHIVE}/targets-created.json" \
+  --fetch-dir "${PRELOAD_DIR}" \
+  --repo-dir "${OSTREE_REPO_DIR}"
```

Comment:
Should we make this configurable/optional for now? Or maybe we go all-in and avoid making us all do factory-config changes to try?

Reply:
It doesn't require any changes to `factory-config` to try it out, those params have default definition at the beginning of the script.

Reply:
but since its not optional, this means that *all* factories will start producing this new content once merged, correct?

Reply:
yes, what should not impact their existing workflow. We can make it optional, but then another question arises - how to do preload?

Reply:
so, I am making it OFF by default, just do exactly as it does now, COMPOSE_APP_USE_OSTREE set in factory-config.yml turns the ostree usage ON.
The only question is in regard to 'fioctl target image' command since it doesn't have access to factory-config.yml.